monitoring/grafana/provisioning/dashboards/dashbaords.yaml

```diff
@@ -0,0 +1,8 @@
+# This config loads all JSON grafana dashboards from the current directory
```

Comment:
Nitpick: the filename has a typo

Reply:
eagle eye :D

Reply:
Added to rest api PR: https://github.com/aragonone/court-backend/pull/132/commits/8828a5a1d12cc67cdb687c6b0cd77f1c8572750a